server/routes/reviews.js

```diff
@@ -1,6 +1,103 @@
 const express = require('express')
 const router = express.Router()
-// const data = require('../data')
-// const userData = data.users
+const { verifyToken } = require('../middleware/auth')
+const { createReview, removeReview, updateReview } = require('../data/reviews')
+const { getUser } = require('../data/users')
+const errors = require('../data/errors')
+const { ObjectId } = require('mongodb')
+let username
+
+router.post('/', verifyToken, async (req, res) => {
+  let { reviewerId, contentId, dateOfReview, review, like_dislike } = req.body
+
+  if (!req.params.id) {
+    throw 'You must specify an ID of the user to post a review'
+  }
+
+  try {
+    const user = await getUser(req.params.id)
+    username = user.username
+
+    reviewerId = errors.validateObjectId(reviewerId)
+    reviewer = errors.validateReviewer(username)
+    contentId = errors.validateObjectId(contentId)
+    dateOfReview = errors.validateDateOfReview(dateOfReview)
+    review = errors.validateReview(review)
+
+    const createdReview = await createReview(
+      reviewerId,
+      reviewer,
+      contentId,
+      dateOfReview,
+      review,
+      like_dislike
+    )
+
+    res.status(200).json(createdReview)
+  } catch (e) {
+    return res.status(400).send(String(e))
+  }
+})
+
+router.put('/', async (req, res) => {
```

Comment:
add the verifyToken middleware to the other reviews route and use that to get the reviewerId